doc-content/enterprise-only/vehicle-route-planning/run-vrp-manually-proc.adoc

```diff
@@ -74,12 +75,14 @@ In the following example, OptaWeb Vehicle Routing downloads the OSM map of Centr
 +
 [source,subs="attributes+"]
 ----
-java -jar optaweb-vehicle-routing-standalone-{MAVEN_ARTIFACT_VERSION}.jar \
---app.persistence.h2-dir=/home/user/.optaweb-vehicle-routing/db \
---app.routing.osm-dir=/home/user/.optaweb-vehicle-routing/openstreetmap \
---app.routing.gh-dir=/home/user/.optaweb-vehicle-routing/graphhopper \
---app.routing.osm-file=central-america-latest.osm.pbf \
---app.region.country-codes=BZ,GT
+java \
+-Dapp.demo.data-set-dir=$HOME/{VRP-DATA-DIR}/dataset \
+-Dapp.persistence.h2-dir=$HOME/{VRP-DATA-DIR}/db \
+-Dapp.routing.gh-dir=$HOME/{VRP-DATA-DIR}/graphhopper \
+-Dapp.routing.osm-dir=$HOME/{VRP-DATA-DIR}/openstreetmap \
+-Dapp.routing.osm-file=entral-america-latest.osm.pbf \
+-Dapp.region.country-codes=BZ,GT
+-jar quarkus-app/quarkus-run.jar
```

Comment:
The command will technically not work because of the missing `\` that should've been added on line 84 (see my previous comment). But it's nothing serious. Just a common typo. If the reader copy-pasted the command they would easily fix the problem.

Reply:
Oh right! Fixed.